pkg/schema/typesystem_validation.go

```diff
@@ -27,7 +35,7 @@ func (def *Definition) Validate(ctx context.Context) (*ValidatedDefinition, erro
 
 		// Validate the usersets's.
 		usersetRewrite := relation.GetUsersetRewrite()
-		rerr, err := graph.WalkRewrite(usersetRewrite, func(childOneof *core.SetOperation_Child) (interface{}, error) {
+		rerr, err := graph.WalkRewrite(usersetRewrite, func(childOneof *core.SetOperation_Child) (any, error) {
```

Comment:
curious, what's the difference between an empty interface and `any`?

Reply:
Nothing; `any` is an alias for `interface{}`, its just the new way of doing so: 

> For convenience, the predeclared type any is an alias for the empty interface.

Reply:
And it's starting to be a lint (at least in my editor) so I just fixed it while I was there